### ISSUE TITLE
Could you provide the processed data for downloading?

### ISSUE THREAD

**Issue by dengyang17** [facebookresearch/ELI5#7]
It takes time to go over all the data creation process. And the slurm cluster is very inconvenient to use. Could you provide the processed data for directly downloading? So much thanks for this if possible.

**Comment by huihuifan** [comment]
thanks for your interest! Unfortunately, due to legal concerns, we cannot release the raw dataset. We are very sorry for this! Please open an issue if you are having trouble downloading the dataset.

**Comment by huihuifan** [comment]
Unclosing so that others can see our response if they have this question

**Comment by ehosseiniasl** [comment]
job 99 is failing with this message,

```2019-08-31 04:40:01 (85.2 MB/s) - ‘processed_data/collected_docs/tmp/CC-MAIN-20180822065454-20180822085454-00719.warc.wet.gz’ saved [105226209/105226209]

Traceback (most recent call last):
  File "download_support_docs.py", line 163, in <module>
    main()
  File "download_support_docs.py", line 63, in main
    fname   = url_lst[i].split('/')[-1][:-3]
IndexError: list index out of range```

**Comment by AmosHua** [comment]
Hi, when was the explainlikeimfive_unigram_counts.json file created during the data creation?  I have never seen it.

**Comment by yjernite** [comment]
Looks like the wet.paths file finished with an empty line, we'll update the script to be robust to that.

In the meantime, looks like you're done downloading, so you can move on to the next step!

**Comment by ehosseiniasl** [comment]
also, 11 jobs are aborting due to time limit! any clue?
`18 22 25 30 36 40 41 70 79 80 90`

error:
`slurmstepd: error: *** JOB 320 ON slrum-cluster-ehsan-compute7 CANCELLED AT 2019-09-02T00:35:39 DUE TO TIME LIMIT ***`

**Comment by yjernite** [comment]
In our experience, some jobs might run significantly slower than others on the cluster is they are assigned to I/O saturated nodes. Check how far along the jobs were (the data size should be roughly similar for all of the downloaded slices), and if they're still missing a lot of pages you can just re-launch them.

You can also edit eli_download_docs.sbatch to request more time

**Comment by ehosseiniasl** [comment]
I increased the request_time to 10 days. Also ran the remaining jobs on gcp. all failed after this time. below is the size of documents for `slice=18`. How can I verify these are all documents for this slice

```  86M  explainlikeimfive/1/docs_slice_00018.json
 104M  explainlikeimfive/2/docs_slice_00018.json
 107M  explainlikeimfive/3/docs_slice_00018.json
  63M  explainlikeimfive/4/docs_slice_00018.json
  62M  explainlikeimfive/5/docs_slice_00018.json
  62M  explainlikeimfive/6/docs_slice_00018.json
  37M  explainlikeimfive/7/docs_slice_00018.json
  36M  explainlikeimfive/8/docs_slice_00018.json
 3.8M  explainlikeimfive/9/docs_slice_00018.json
 5.8M  explainlikeimfive/j/docs_slice_00018.json
 1.8M  explainlikeimfive/k/docs_slice_00018.json
 1.9M  explainlikeimfive/l/docs_slice_00018.json
 1.7M  explainlikeimfive/m/docs_slice_00018.json
 2.0M  explainlikeimfive/n/docs_slice_00018.json
 2.4M  explainlikeimfive/o/docs_slice_00018.json
 1.5M  explainlikeimfive/p/docs_slice_00018.json
 2.9M  explainlikeimfive/q/docs_slice_00018.json
 1.6M  explainlikeimfive/r/docs_slice_00018.json
 1.6M  explainlikeimfive/s/docs_slice_00018.json
 1.6M  explainlikeimfive/t/docs_slice_00018.json
 1.5M  explainlikeimfive/u/docs_slice_00018.json
 1.1M  explainlikeimfive/v/docs_slice_00018.json
 1.5M  explainlikeimfive/w/docs_slice_00018.json
 1.9M  explainlikeimfive/x/docs_slice_00018.json
 2.0M  explainlikeimfive/y/docs_slice_00018.json
 2.8M  explainlikeimfive/z/docs_slice_00018.json```

**Comment by ehosseiniasl** [comment]
my final splits file-sizes

**Comment by yjernite** [comment]
Responding after a while sorry for the delay!

The time-outs were cause by an issue with the Spacy tokenizer, we fixed these and the threads are now much more likely to finish in under 24 hours.

Also, your files are about twice as large as the ones we get from running the scripts (final sizes have been added to the README.md). Since some of the process structure has been changed to improve robustness, you would have to re-run steps 2 and 3.

**Comment by santhoshkolloju** [comment]
can some please share the google drive link / s3 bucket with this extracted data.would be very helpful, As i am not able to complete this download on my mac.

**Comment by mahmuduzzaman** [comment]
At this stage ./eli_download_docs_launcher.sh

i am getting the following error
sbatch: error: This does not look like a batch script.  The first
sbatch: error: line must start with #! followed by the path to an interpreter.
sbatch: error: For instance: #!/bin/sh

in ubuntu system for windows
Yes it is becoming extremely hard to find the file. could you please share the files

**Comment by swaroop-nath** [comment]
What is the estimated time to recreate this data? I request this info to estimate the costs I would incur. It would also be beneficial if people provided the kinds of instances they used to recreate the data.
